wikimedia/wikipedia-ios
# Native Page Editor Container and UI Tests

**Phabricator:** https://phabricator.wikimedia.org/T347870

### Notes
* This PR brings over to the main app the contents of this PR https://github.com/wikimedia/wikipedia-ios-components/pull/9
* It also adjusts UI Tests to run in isolation from unit tests

### Test Steps
1. To test the component itself, run the app On the staging target and tap edit on an article. You should see the new editor
2. To run the UI Tests, hit CMD +U on the new WikipediaUITests target
3. Run the tests on  Xcode Cloud - manually trigger the UI Tests workflow under the main app configurations

Note: currently the theming is not working on the test plan configurations, I'm investigating that
